static/js/shared.js

```diff
@@ -137,9 +137,32 @@ window.onload = function checkBrowser(){
             if (scroll >= 100) sticky.addClass('.notice');
             else sticky.removeClass('.notice');
           });
+     }
+}
+
+function checkVersions(){
+    function evaluateVersions(data){
+        console.log(data)
```

Comment:
remove log and comment

Reply:
already removed locally, the function itself needs some work